primefaces/src/main/resources/META-INF/resources/primefaces/fileupload/2-fileupload.js

```diff
@@ -717,6 +717,9 @@ PrimeFaces.widget.FileUpload = PrimeFaces.widget.BaseWidget.extend({
             var update = PrimeFaces.expressions.SearchExpressionFacade.resolveComponents(this.jq, this.cfg.update).join(' ');
             PrimeFaces.ajax.Request.addParam(params, PrimeFaces.PARTIAL_UPDATE_PARAM, update, parameterPrefix);
         }
+        if (this.cfg.ignoreAutoUpdate) {
+            PrimeFaces.ajax.Request.addParam(params, PrimeFaces.IGNORE_AUTO_UPDATE_PARAM, true, parameterPrefix);
```

Comment:
could we just reuse the whole createFacesAjaxFormData method?

Reply:
one if FormData and one is ParamData.  They definitely look different?

Reply:
maybe you can create a refactoring task if we can reuse it

Reply:
I will add a task to the other ticket I created about AjaxSource refactoring.